src/lib.rs

```diff
@@ -281,6 +281,20 @@ impl Url {
     }
 
     /// Return a default `ParseOptions` that can fully configure the URL parser.
+    ///
+    /// # Examples
+    ///
+    /// Get default `ParseOption`, then change base url
+    ///
+    /// ```rust
+    /// use url::Url;
+    ///
+    /// let options = Url::options();
+    /// let api = Url::parse("https://api.example.com").ok();
+    /// let base_url = options.base_url(api.as_ref());
```

Comment:
Actually, according to the other examples (and as highlighted by @dtolnay), we should not be using `unwrap()` either. Let's use `?` instead, thus forcing users to handle the errors properly.